<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDateEdit{

border:none;
border-radius:10px;
color:white;
background:#8b0000;
font-family: &quot;Roboto&quot;,sans-serif;

font-size:5px;
font-weight:lighter;



border-top-right-radius: 6px;
border-bottom-right-radius: 30px;



    font-size: 15px;

    text-align: center;

    margin: 5px;
  font-family: &quot;Roboto&quot;,sans-serif;

}

QGroupBox{
border:2px solid #8b0000;
border-radius:70px;


    font-size: 15px;
    color: white;
    text-align: center;
    font-weight: bold;
    margin: 5px;
  font-family: &quot;Roboto&quot;,sans-serif;



}

QLabel{
    font-size: 13px;
   color:#c70000;
    text-align: center;
    font-weight: 600;
    margin: 5px;
    
    font-family: &quot;Roboto&quot;,sans-serif;
}

QLineEdit{
outline: none;
   border:none;
font-family: &quot;Roboto&quot;,sans-serif;
color: white;
font-size:15px;
font-weight: 200;
font-weight:bold;
    padding:20px 10px;
    
    width: 20%;
    background: transparent;

}


QPushButton{
color:white;
font-weight:bold;
font-size:13px;
background-color:transparent;
border:2px solid #8b0000;
border-radius:15px;



}






QPushButton:hover{
color:white;
background:#8b0000;
border:2px solid white;
border-radius:15px;



}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
	border-image: url(:/images/login3.jpeg);
	background-size:cover;


}</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>391</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox{
font-size:25px;
}</string>
    </property>
    <property name="title">
     <string>LogIn</string>
    </property>
    <widget class="QLineEdit" name="login_username">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>i.e. Jon</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label{
font-size:15px;
}
</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_2{
font-size:15px;
}

</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_password">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>*****</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>350</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>80</y>
      <width>431</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox_2{
font-size:25px;
}</string>
    </property>
    <property name="title">
     <string>Sign Up</string>
    </property>
    <widget class="QLineEdit" name="firstname_sign">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string>e.g. Arpil</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lastname_sign">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>e.g. Barua</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>First Name (username)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Birth Date</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="bdate_sign">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#F0F0F0;
color:#8b0000;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Blood Group</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="blood_group_sign">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>320</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>e.g. A+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>360</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone No.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="phone_sign">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>390</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>+8801851-**133*</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>440</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Password</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>340</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>410</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_sign">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>460</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>480</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>520</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Open Account</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
